Doc/library/typing.rst

```diff
@@ -428,12 +428,12 @@ value of type :data:`Any` and assign it to any variable::
 
    from typing import Any
 
-   a = None    # type: Any
-   a = []      # OK
-   a = 2       # OK
+   a: Any = None
+   a = []          # OK
+   a = 2           # OK
 
-   s = ''      # type: str
-   s = a       # OK
+   s: str = ''
```

Comment:
I don't really like this example. Because it is quite a common mistake to make (especially for new-to-typing developers): adding type annotations to variables where type is inferred. 

So, maybe we can use a comment with something like `# The type is "str"`?

Reply:
Hmm, I see your point

Reply:
How's it now?